src/nvim/indent_c.c

```diff
@@ -513,23 +513,31 @@ static int cin_isdefault(const char_u *s)
 }
 
 /*
- * Recognize a "public/private/protected" scope declaration label.
+ * Recognize a scope declaration label set in 'cinscopedecls'.
```

Comment:
Comment style needs to be changed to `///`-style. Since this comment change was not in the patches, wait for <https://github.com/vim/vim/pull/10126> first.

Reply:
Done